recipes/pocketsphinx/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "pocketsphinx-python" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/bambocher/{{ name }}/archive/v{{ version }}.tar.gz
+    sha256: d56244340161847cd978fdf9aba0d477bf5f99fcb3019f5b10dea921d078e23a 
+  - git_url: https://github.com/cmusphinx/pocketsphinx
+    git_tag: 1fdc9ccb637836d45d40956e745477a2bd3b470a
+    folder: deps/pocketsphinx
+  - git_url: https://github.com/cmusphinx/sphinxbase
```

Comment:
Please do not use git for your source, see https://conda-forge.org/docs/meta.html#build-from-tarballs-not-repos

Reply:
How should I handle this situation if the upstream repo has not made a proper release?  I see their code on SurgeForce as well, but I'm not sure what version there matches to the commits of the packages under `deps`.

Reply:
I'll break this into separate recipes.   Thank you for your help!

Reply:
No problem !